doc/whatsnew/3/3.3/index.rst

```diff
@@ -14,6 +14,54 @@ Summary -- Release highlights
 
 .. towncrier release notes start
 
+What's new in Pylint 3.3.5?
+---------------------------
+Release date: 2025-03-08
```

Comment:
```suggestion
Release date: 2025-03-08
```
Date might need to change as we need to fix https://github.com/pylint-dev/pylint/actions/runs/13742008950/job/38432352767 first

Reply:
Fixed https://github.com/pylint-dev/pylint/actions/runs/13742008950/job/38432596625